pkgs/development/libraries/dbus/default.nix

```diff
@@ -3,6 +3,7 @@
 , fetchurl
 , pkgconfig
 , expat
+, enableSystemd ? stdenv.isLinux && !stdenv.hostPlatform.isMusl
 , systemd
```

Comment:
Should I set `systemd` to `systemd ? null` as it is done for libusb1?

https://github.com/NixOS/nixpkgs/pull/71563/files#diff-81a3f0c1548ebc7ea522eb8c443aaa01R5